verifiers/internal/gha/builder.go

```diff
@@ -19,8 +19,9 @@ var (
 )
 
 var defaultTrustedReusableWorkflows = map[string]bool{
-	trustedBuilderRepository + "/.github/workflows/generator_generic_slsa3.yml": true,
-	trustedBuilderRepository + "/.github/workflows/builder_go_slsa3.yml":        true,
+	trustedBuilderRepository + "/.github/workflows/generator_generic_slsa3.yml":   true,
```

Comment:
we may bucket these into container and not; and verify that the trusted builder is as expected (container vs artifact)

Reply:
Good call!

Reply:
curious: do you think this should be captured in BuilderOpts (internally generated depending on whether the artifact was a blob or container image)

Reply:
i guess we could. But it's probably sufficient to select the correct "bucket" based on which function we're executing (`VerifyContainer` vs `VerifyArtifact`)?

Reply:
yeah, that makes sense.